utils/_context/_scenarios/core.py

```diff
@@ -310,13 +312,24 @@ def _get_warmups(self):
         if not self.replay:
             warmups.append(create_network)
 
-            for container in self._required_containers:
-                warmups.append(container.start)
+        warmups.append(self._start_containers)
```

Comment:
Containers must not be started in replay mode

Reply:
They aren't, there is a check in `start_container`.

Reply:
Ok by reading the code, I understood that you also want to have the `post_start` in the threads, am I right ? 

Could you just add a comment explaining why, it'll save some cognitive effort for future maintainers 🙏 .

Reply:
I can add a comment but I'm not sure I understand the part that needs a comment. Can you clarify what needs to be clarified? 😆

Reply:
The purpose of `replay` mode is to no start anything. By just reading this code, we cannot understand why `_start_containers` is called on replay mode.

```suggestion
        # on replay mode, _start_containers() won't actually start containers, 
        # but will only call the necessary post_start method
        warmups.append(self._start_containers)
```

Reply:
I tend to prefer more descriptive code to comment, maybe I could name the method something better 🤔